src/sentry/incidents/subscription_processor.py

```diff
@@ -582,21 +590,36 @@ def trigger_alert_threshold(
                 },
             )
             return None
+        # 'threshold_period' - how many times an alert value must exceed the threshold to fire/resolve the alert
         if self.trigger_alert_counts[trigger.id] >= self.alert_rule.threshold_period:
             metrics.incr("incidents.alert_rules.trigger", tags={"type": "fire"})
 
             # Only create a new incident if we don't already have an active one
             if not self.active_incident:
                 detected_at = self.calculate_event_date_from_update_date(self.last_update)
+                activation: AlertRuleActivations | None
+                if self.alert_rule.monitor_type == AlertRuleMonitorType.ACTIVATED:
+                    activations = list(self.subscription.alertruleactivations_set)
+                    if len(activations) > 1 or len(activations) == 0:
```

Comment:
nit: simplify logic to one statement
```suggestion
                    if len(activations) >= 0:
```

Reply:
Ah - slightly different.

I could change it to `len(activations) != 1` though